src/transactions/FeeBumpTransactionFrame.cpp

```diff
@@ -321,11 +321,26 @@ FeeBumpTransactionFrame::getFullFee() const
     return mEnvelope.feeBump().tx.fee;
 }
 
+int64
+FeeBumpTransactionFrame::declaredSorobanResourceFee() const
```

Comment:
Where are we using this?

Reply:
in `"negative inclusion fee"` test case. it's not used much, but it seems harmless to have this helper, so callers don't need to know the internals of fee bumps to extract the inner tx's resource fee.

Reply:
If that's test-only, then I would suggest to make it public (and part of the interface) only for tests - so far there wasn't an intent/need to access soroban resource fee outside of the tx frame itself.